src/cmd/terraform.go

```diff
@@ -310,30 +310,49 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 func exportTeams(c *opslevel.Client, config *os.File, shell *os.File) {
 	shell.WriteString("# Teams\n")
 
-	teamConfig := `resource "opslevel_team" "%s" {
-  name = "%s"
-  manager_email = "%s"
-  parent = "%s"
-  %s
-  %s
+	teamConfig := `resource "opslevel_team" "%s" {%s
 }
+
 `
 	resp, err := c.ListTeams(nil)
 	teams := resp.Nodes
 	cobra.CheckErr(err)
 	for _, team := range teams {
+		teamBody := ""
+
 		aliases := flattenAliases(team.Aliases)
 		if len(aliases) > 0 {
```

Comment:
As of [12 days ago](https://github.com/OpsLevel/terraform-provider-opslevel/pull/157/files#diff-c294bb8f5a3b817c0ee0a0e6054a395f6fde4d7cd4a9f8d55ff29700db4c4054R43) team `aliases` are required so this `len()` check should be removed.